ger_deps

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -e
+
+cd
+sudo rm -rf /tmp/installitrude/
+mkdir /tmp/installitrude
+cd /tmp/installitrude
+clear
+
+# INSTALLATION OF LUAROCKS
+wget https://luarocks.org/releases/luarocks-3.9.2.tar.gz
```

Comment:
that good but should only be done if luarocks not already installed, since you install a specific version of it which we will probably not change and can mess with the user's previous install if already present on the system.

Reply:
just run luarocks and see if it's alive I guess